querybook/webapp/ui/Modal/types.ts

```diff
@@ -1,9 +1,14 @@
+import React from 'react';
+
 interface IBaseModalProps {
     hideModalTitle?: boolean;
     className?: string;
 
     // defaults to '', set to null to hide title
     title?: string;
+    infoDOM?: React.ReactElement;
```

Comment:
to keep it simple, i think we can use topDOM to replace infoDOM, they can just supply their own title via <Title> component

Reply:
otherwise infoDOM is only rendered if title is provided